schemas/oic.wk.mnon-schema.json

```diff
@@ -0,0 +1,72 @@
+{
+  "id": "http://openinterconnect.org/schemas/oic.wk.nmon#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description" : "Copyright (c) 2017 Open Connectivity Foundation, Inc. All rights reserved.",
+  "title": "network monitoring",
+  "definitions": {
+    "nmon": {
+        "network": 
```

Comment:
rename to iana-interfacetype.

https://www.iana.org/assignments/ianaiftype-mib/ianaiftype-mib